addons/sale_project/models/project_update.py

```diff
@@ -0,0 +1,17 @@
+from odoo import api, models
+
+
+class ProjectUpdate(models.Model):
+    _inherit = 'project.update'
+
+    @api.model
+    def _get_template_values(self, project):
+        template_values = super()._get_template_values(project)
+        profitability_values = template_values.get('profitability')
+        if profitability_values and 'revenues' in profitability_values and 'data' in profitability_values['revenues']:
+            for lines in profitability_values['revenues']['data']:
```

Comment:
nitpick, but i'd rename 'lines' into 'section'. Since we're fetching sales order lines on the next code line, it could be confusing.